Cpp_hands_on_programming/lib/shm/Ipc_shm.cc

```diff
@@ -124,8 +124,8 @@ void Ipc_shm::receive() {
 }
 
 void Ipc_shm::check_leadingSlash() {
```

Comment:
should be removed?

Reply:
Right it is supposed to be, but there are too many commented out code in shm, so I decided to leave it there just for this story(QA 473). Unrelated code will be removed properly for error handling in the next story (QA 189)